File: src/main/kotlin/com/chriskelly/kotlincourse/interop/ICustomerRepository.java
package com.chriskelly.kotlincourse.interop;

import com.chriskelly.kotlincourse.classes.CustomerJava;

import java.util.List;

/**
 * ICustomerRepository
 *
 * @author bigkahuna
 * @since 26/03/2017
 */
public interface ICustomerRepository {

    CustomerJava getById(int id);

    List<CustomerJava> getAll();
}

File: src/main/kotlin/com/chriskelly/kotlincourse/interop/ICustomerRepository.kt
package com.chriskelly.kotlincourse.interop

import com.chriskelly.kotlincourse.classes.CustomerJava

/**
 * ICustomerRepository

 * @author bigkahuna
 * *
 * @since 26/03/2017
 */
interface ICustomerRepository {

    fun getById(id: Int): CustomerJava

    val all : List<CustomerJava>
}
